test/unit/behaviours/tenancy-start-west-midlands-check.test.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const reqres = require('reqres');
+const Behaviour = require('../../../apps/right-to-rent-check/behaviours/tenancy-start-west-midlands-check');
+
+describe('apps/behaviours/tenancy-start-west-midlands-check', () => {
+  it('exports a function', () => {
+    expect(Behaviour).to.be.a('function');
+  });
+
+  class Base {
+      getNextStep() {}
+      saveValues() {}
+    }
+    let req;
+    let res;
+    let sessionModel;
+    let instance;
+    let ValidTenants;
+
```

Comment:
yeah, `class` in in the correct position, but it's definition and all the `let`'s are two spaces too far to the right